packages/host/app/components/operator-mode/container.gts

```diff
@@ -467,8 +478,13 @@ export default class OperatorModeContainer extends Component<Signature> {
         return await newItem.request?.promise;
       },
       viewCard: async (card: CardDef, format: Format = 'isolated') => {
+        let liveCard = await here.cardService.loadModel(here, new URL(card.id));
+        if (!liveCard) {
+          throw new Error(`bug: could not load card ${card.id}`);
+        }
         here.addToStack({
-          card,
+          // assert that we are using a live card
+          card: liveCard,
```

Comment:
this change ensures that the card we are viewing is the live card--which also exposed the bug that the inner object identities (field values) of the `card-api.updateFromSerialized()` was not preserving object identities all the way down (our existing operator mode integration tests revealed this issue).